public_html/wp-content/plugins/wcpt/wcpt-event/class-event-application.php

```diff
@@ -67,7 +67,26 @@ public function render_application_shortcode() {
 			$this->submit_application( $_POST );
 		} else {
 			$countries = wcorg_get_countries();
-			$this->render_application_form( $countries );
+
+			$wporg_username = '';
+			if ( is_user_logged_in() ) {
+				$current_user = wp_get_current_user();
+				$prefilled_fields = array(
+					'wporg_name'			=> $current_user->display_name,
+					'wporg_username'	=> $current_user->user_login,
+					'wporg_email'			=> $current_user->user_email,
+				);
+			}
+
+			if ( ! is_user_logged_in() ) {
+				echo '<div class="wcfd-disabled-form">' . wcorg_login_message( '', get_permalink() ) . '<div class="wcfd-overlay"></div><div inert>';
```

Comment:
```suggestion
				// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
				echo '<div class="wcfd-disabled-form">' . wcorg_login_message( '', get_permalink() ) . '<div class="wcfd-overlay"></div><div inert>';
```

This "security issue" can be ignored, we can add better escaping into #524.